CMakeLists.txt

```diff
@@ -5,13 +5,19 @@ project( CroftEngine VERSION 1.8 )
 
 file( STRINGS deprecations.txt _deprecations )
 foreach( _deprecation IN LISTS _deprecations )
-    if( NOT _deprecation MATCHES ".+ .+" )
+    if( _deprecation STREQUAL "" )
```

Comment:
Sorry I missed this. 
`if( NOT _deprecation MATCHES ".+ .+" )` would have skipped lines in the correct format.

Reply:
The previous version just skipped every invalid line and silently swallowed it. I just took the opportunity to add some additional validation.